src/raft.rs

```diff
@@ -272,20 +278,15 @@ impl<T: Storage> Raft<T> {
                 priority: c.priority,
             },
         };
-        for p in voters {
-            let pr = Progress::new(1, r.max_inflight);
-            if let Err(e) = r.prs.insert_voter(*p, pr) {
-                fatal!(r.logger, "{}", e);
-            }
-            if *p == r.id {
-                r.promotable = true;
-            }
-        }
-        for p in learners {
-            let pr = Progress::new(1, r.max_inflight);
-            if let Err(e) = r.prs.insert_learner(*p, pr) {
-                fatal!(r.logger, "{}", e);
-            };
+        confchange::restore(&mut r.prs, r.r.raft_log.last_index(), conf_state)?;
+        let new_cs = r.post_conf_change();
```

Comment:
Rust uses abbreviation as well. For example, you can see `Vec` and `U64` instead of `Vector` and `UnsignedInt64`. We generally follows both the style of [Rust](https://github.com/rust-dev-tools/fmt-rfcs/blob/master/guide/guide.md) and https://pingcap.github.io/style-guide/rust/.

Reply:
I think `post_conf_change` is better because it's actually doing some side effects to `r` **after** `r.prs` restored from confstate